cloud/google/configure_service.sh

```diff
@@ -34,61 +34,120 @@ ZONE_LIST=$(echo ${NODE_ZONES[*]} | tr " " "\n" | sort | uniq )
 
 echo "============================================================="
 echo "Creating network rules..."
-gcloud compute http-health-checks create get-sth-check \
-    --port 80 \
-    --request-path /ct/v1/get-sth
-gcloud compute firewall-rules create ${INSTANCE_TYPE}-node-80 \
-    --allow tcp:80 \
-    --target-tags ${INSTANCE_TYPE}-node
+echo "  http-health-checks create..."
+PRESENT=`gcloud compute http-health-checks list get-sth-check | grep get-sth-check`
+if [ "$PRESENT" == "" ]; then
+  gcloud compute http-health-checks create get-sth-check \
+      --port 80 \
+      --request-path /ct/v1/get-sth
+else
+  echo "  ...get-sth-check already present"
+fi
+echo "  firewall-rules create..."
+PRESENT=`gcloud compute firewall-rules list ${INSTANCE_TYPE}-node-80 | grep ${INSTANCE_TYPE}-node-80`
```

Comment:
might be worth pulling the name of the item out in all of if-not-present-then-create blocks to guard against typos?

```bash
THING="${INSTANCE_TYPE}-node-80"
PRESENT=`... list ${THING} | grep ...`
if [ "${PRESENT}" == "" ]; then
  ... create ${THING} ...
  ...
fi
...
```